trafficlight/internals/client.py

```diff
@@ -255,3 +256,6 @@ async def advance_clock(self, duration: int) -> None:
         await self._perform_action(
             {"action": "advance_clock", "data": {"milliseconds": duration}}
         )
+
```

Comment:
I can't see where go_offline is being used - do we need this still - is the test OK with using the network_proxy to make the client be offline?

Reply:
(this selected the wrong bit of code; the code in question is just below this line; sorry)